packages/enzyme-test-suite/test/ShallowWrapper-spec.jsx

```diff
@@ -558,18 +558,17 @@ describe('shallow', () => {
       expect(wrapper.find('.row + .row')).to.have.lengthOf(1);
     });
 
-    it('should error sensibly if prop selector without quotes', () => {
+    it('should throw for non-numeric attribute values without quotes', () => {
       const wrapper = shallow(
         <div>
           <input type="text" />
           <input type="hidden" />
+          <input type="text" />
         </div>,
       );
-
-      expect(() => wrapper.find('[type=text]')).to.throw(
-        TypeError,
-        'Enzyme::Unable to parse selector \'[type=text]\'. Perhaps you forgot to escape a string? Try \'[type="text"]\' instead.', // eslint-disable-line max-len
-      );
+      expect(() => wrapper.find('[type=text]')).to.throw();
```

Comment:
can we add more info for all these `throw` assertions? it'd be nice to assert at least on the exception type; even nicer to assert on the error message